.github/workflows/gradle-check.yml

```diff
@@ -0,0 +1,73 @@
+name: Jenkins Gradle Check
+on: [pull_request]
```

Comment:
Do we need to include `push` ?

Reply:
No need. Every time there is a new commit it will auto trigger again as long as pull_request is the trigger.